src/CONST.js

```diff
@@ -670,6 +670,7 @@ const CONST = {
             FREE: 'free',
             PERSONAL: 'personal',
             CORPORATE: 'corporate',
+            TEAM: 'team',
```

Comment:
Unrelated to revert. I saw a console error about this and decided to patch it up. Ideally, I think we should have this entire [list](https://github.com/Expensify/Web-Expensify/blob/e8c795f5ff01a8627b71efb90c39865acaa033a5/lib/Policy.php#L33-L99) over here.